### PR TITLE
feat: theme adjust shadow values

### PR DESCRIPTION
**Pull request by kimak**
## Checklist before requesting a review
- [x] Working on iOS
- [x] Working on Android
- [x] Visual regressions screenshots are up to date
- [x] Design validated

### CODE REVIEW

File: packages/sandbox/e2e/android/screenshots/Shadows.png
Comment (kimak) [inline packages/sandbox/e2e/android/screenshots/Shadows.png:1]: @hcourthias can you check if you have the same rendering than me on `Pixel5 API 33` please?

File: packages/sandbox/src/app/sandbox/docs/shadowsValidation.doc.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+            onPress: () => {
+              console.log('Primary button pressed');
+            },
+            appearance: 'neutral',
+          }}
+        />
+      </Box>
+    </Box>
+  );
+};
```
Comment (kimak) [inline packages/sandbox/src/app/sandbox/docs/shadowsValidation.doc.tsx]: @ianrobertlambert did you have time to review shadows on your devices so I can delete this examples?
Screenshot 2023-04-25 at 18 40 32

https://qr.expo.dev/eas-update?projectId=11eb7a11-ac37-40c7-a649-095da5a7b0b3&groupId=b5e8e929-d2aa-46e7-81a5-54aeaa8322b2

File: packages/sandbox/e2e/ios/screenshots/ButtonDock.png
Comment (ludwig-pro) [inline packages/sandbox/e2e/ios/screenshots/ButtonDock.png:1]: pourquoi ça change ?

**Review by ludwig-pro — APPROVED** [review]

File: packages/sandbox/e2e/ios/screenshots/ButtonDock.png
Comment (hcourthias) [reply packages/sandbox/e2e/ios/screenshots/ButtonDock.png:1]: erreur de merge

File: packages/sandbox/e2e/android/screenshots/Shadows.png
Comment (hcourthias) [reply packages/sandbox/e2e/android/screenshots/Shadows.png:1]: je check

**Review by hcourthias — APPROVED** [review]

**Pull request merged**